woocommerce/class-sv-wc-helper.php

```diff
@@ -1041,16 +1041,15 @@ public static function is_enhanced_admin_screen() {
 	 *
 	 * @since 5.10.6
 	 *
+	 * @deprecated with no alternatives
+	 *
 	 * @return bool
 	 */
-	public static function is_wc_navigation_enabled() {
-
-		return
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Navigation\Screen::class, 'register_post_type' ] ) &&
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Navigation\Menu::class, 'add_plugin_item' ] ) &&
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Navigation\Menu::class, 'add_plugin_category' ] ) &&
-			is_callable( [ \Automattic\WooCommerce\Admin\Features\Features::class, 'is_enabled' ] ) &&
-			\Automattic\WooCommerce\Admin\Features\Features::is_enabled( 'navigation' );
+	public static function is_wc_navigation_enabled() : bool
+	{
+		error_log('The Enhanced navigation feature has been deprecated since WooCommerce 9.3 with no alternative. Navigation classes will be removed in WooCommerce 9.4');
+
+		return false;
```

Comment:
Do you think we need a Woo version check here? [ref](https://godaddy.slack.com/archives/C01NDQZULKF/p1727279715494199?thread_ts=1727204166.496609&cid=C01NDQZULKF)

As in:

If on Woo GTE 9.3, then return false like you're doing here.
Otherwise, use previous code.

Reply:
I can't think of any benefit for keeping the integration in the first place as it is already deprecated and will be removed pretty soon. I am leaning towards removing toward always returning false